Modelica_DeviceDrivers/Resources/Include/MDDRealtimeSynchronize.h

```diff
@@ -959,15 +963,16 @@ DllExport void MDD_RTSyncSynchronize(void * rtSyncObj, double simTime, double sc
         /* wait until (scaled) simulation time == real-time */
         ret = clock_nanosleep(CLOCK_MONOTONIC, TIMER_ABSTIME, &t_abs, NULL);
         if (ret) {
-            ModelicaError("MDDRealtimeSynchronize.h: clock_nanosleep(..) failed\n");
+            ModelicaFormatError("MDDRealtimeSynchronize.h: clock_nanosleep(..) failed\n");
```

Comment:
That change seems undesired, since the first argument is not a format string.

Reply:
Yes, I've had quite some "printf" debugging before finding the cause and this as well as the line below are artifacts from the debugging... I will remove it.